frontend/src/routes/ServicesQueues.js

```diff
@@ -1,87 +1,141 @@
-import React, { useState } from "react"
-import PropTypes from "prop-types"
+import React from "react"
```

Comment:
@hhummel unless there is something specifically in this file you want me to look at I am not going to look at the changes here since this is stuff that Robby is working on and I have already changed this file a lot on a separate branch

Reply:
I'm surprised to see getting rid of useState...